cfpb/cfgov-refresh
# Update MP front-end to use cfpb-chart-builder v3.x

Updates the chart builder dep to v3 and changes paths accordingly. Also adds a stylesheet and scaffolds out a dynamic SVG version of the map's legend.

## Additions

- A stylesheet (`mortgage-performance-trends.less`) for the mortgage performance chart blocks.

## Removals

- `cfpb-chart-builder-canary` dependency which was a placeholder for the chart library's dev branch.

## Changes

- Upgrade `cfpb-chart-builder` to v3 which includes the new Mortgage Performance chart types.
- Minor template and JS changes to accommodate v3.
- Convert the map's legend from a static image to a dynamic SVG.

## Testing

1. Pull down this branch and `./setup.sh`
1. `./runserver.sh` and the [CCT pages](http://localhost:8000/data-research/consumer-credit-trends/auto-loans/lending-neighborhood-income-level/) should all look normal.

To test the new MP charts you'll need to:

1. Set up the [mortgage performance API](https://github.com/cfpb/cfgov-refresh/pull/3189).
1. Create a [mortgage chart block and page](https://github.com/cfpb/cfgov-refresh/pull/3260).

## Checklist
